bin/handler-slack.rb

```diff
@@ -114,9 +129,13 @@ def post_data(notice)
     http.use_ssl = true
 
     req = Net::HTTP::Post.new("#{uri.path}?#{uri.query}", 'Content-Type' => 'application/json')
-    text = slack_surround ? slack_surround + notice + slack_surround : notice
 
-    req.body = payload(text).to_json
+    if payload_template.nil?
+      text = slack_surround ? slack_surround + notice + slack_surround : notice
+      req.body = payload(text).to_json
+    else
+      req.body = notice
```

Comment:
I can't change the variable name from "notice" to "body" if that makes more sense.